view_user_birthdate.py

```diff
@@ -0,0 +1,242 @@
+"""Build a dictionary of usernames and their corresponding birthdates
+formatted to match their respective nationalities. Additionally, take
+in user input, match it to any usernames in the dictionary and return
+that user's formatted birthdate.
+"""
+import re
+from get_input import get_input
+
+class User(object):
+    """ Take in two strings 'username' and 'nationality' and a Date
+    'birthdate'. Create an object of type User with those same
+    variables as attributes. Keep track of all instantiated User
+    objects with the dictionary 'bday_book'.
+    """
+    bday_book = {}
+
+    def __init__(self, username, nationality, birthdate):
```

Comment:
research the __str__ and __repr__ magic methods in python, and use those to print your user rather than passing strings around.  this will allow you to pass around your User objects, but when you do print User('a','b','c') it will give back a formatted print string however you like

Do the same with date, *put the call to format in the __repr__ method*